projects/storefrontlib/src/shared/components/media/media.service.ts

```diff
@@ -53,6 +54,7 @@ export class MediaService {
     return {
       src: this.resolveAbsoluteUrl(mainMedia?.url),
       alt: alt || mainMedia?.altText,
+      role: role || mainMedia?.role,
```

Comment:
`or` operator treat empty string as false. Do we want to allow override it to `''`? If yes, lets use `??`
```suggestion
      alt: alt ?? mainMedia?.altText,
      role: role ?? mainMedia?.role,
```